.github/workflows/ci-build-proto.yml

```diff
@@ -14,30 +14,92 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        # We only test on the oldest version we want to support and latest.
+        # We trust that things also work for versions in the middle.
```

Comment:
That's brave ;)

Reply:
A lot less brave than what we have today, which is to only test a single version.